### PR TITLE
added connection-test option

### PR DESCRIPTION
**Pull request by yogthos**
I rand into an issue using JTDS MS SQL driver where I need to explicitly disable the connection test. Added the missing option.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,6 +41,7 @@ You'll also need to add the JDBC driver needed for your database.
...
 | `:server-name`              | No       | Depends on the adapter | This property sets the hostname client connects to.                                                                                                                                                                                                                                                                                    |
 | `:port-number`              | No       | Depends on the adapter | This property sets the port clients connects on.                                                                                                                                                                                                                                                                                       |
+| `:connection-test`          | No       | None                   | This property will toggle whether connection test is performed, must be explicitly set to `false` in order to disable the test. |
```
Comment (tomekw) [inline README.md]: Can we assume it's `true` by default? I always prefer to be explicit :)

**Comment by tomekw** [comment]
Thank you @yogthos! Could you also update the CHANGELOG, please? :)

**Comment by yogthos** [comment]
👍

File: README.md (hunk 1 above)
Comment (tomekw) [reply README.md]: @yogthos WDYT about this one? :)

Comment (yogthos) [reply README.md]: Don't have a strong feeling on this one. :) It's an optional field in hikari config, so it ends up defaulting to true there when not set.

**Comment by tomekw** [comment]
Thanks! I will try to release it today!

**Comment by yogthos** [comment]
Fantastic, thanks for the quick turnaround. :)

**Pull request merged**